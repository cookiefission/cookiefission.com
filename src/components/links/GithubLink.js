import React from 'react';
import IconLink from './IconLink';

const GithubLink = () => {
  return(
    <IconLink href='https://github.com/cookiefission' iconName='github' />
  )
};

export default GithubLink;
