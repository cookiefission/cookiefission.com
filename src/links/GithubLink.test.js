import React from 'react';
import GithubLink from './GithubLink';
import renderer from 'react-test-renderer';

describe('<GithubLink />', () => {
  it('matches the snapshot', () => {
    const component = renderer.create(
      <GithubLink />
    );

    expect(component.toJSON()).toMatchSnapshot();
  });
});
