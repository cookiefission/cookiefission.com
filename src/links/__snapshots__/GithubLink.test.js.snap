exports[`<GithubLink /> matches the snapshot 1`] = `
<a
  className="IconLink"
  href="https://github.com/cookiefission"
  target="blank">
  <div
    className="Icon-wrapper">
    <i
      className="fa fa-github fa-lg" />
  </div>
</a>
`;
